src/nadzoring/plugins/connectors/arp.py

```
@@ -0,0 +1,117 @@
+
+        try:
+            cache = ARPCache()
+            entries = cache.get_cache()
+            return ProbeResult(
+                status="ok",
+                details={
+                    "data": [e.__dict__ if hasattr(e, "__dict__") else e for e in entries],
+                    "count": len(entries),
+                },
+            )
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)